CHANGELOG.md

```diff
@@ -78,6 +78,10 @@ Version 9.14.0 adds Apache Kafka instrumentation for the rdkafka and ruby-kafka
 
   To address a recent Bundler deprecation warning, we started using `Bundler.rubygems.installed_specs` instead of `Bundler.rubygems.all_specs` in environments that seemed appropriate. We discovered the version constraint we used was too low. Now, rather than check the version, we check for the method using `respond_to?`. [PR#2853](https://github.com/newrelic/newrelic-ruby-agent/pull/2853)
 
+- **Bugfix: Support view_component v3.15.0+**
```

Comment:
Should we list this one on the top summary too?

Reply:
Sure, added in [ad5f9b8](https://github.com/newrelic/newrelic-ruby-agent/pull/2873/commits/ad5f9b8ee769cc15c01751d21ec2fbbee6b2d724)